.github/workflows/main.yml

```diff
@@ -0,0 +1,53 @@
+name: ci
```

Comment:
fixed some bugs in the gh actions please refresh this file with https://github.com/ipfs/aegir/wiki/Github-Actions-Setup